generators/csharp/codegen/src/ast/Method.ts

```diff
@@ -86,11 +91,34 @@ export class Method extends AstNode {
     public write(writer: Writer): void {
         if (this.summary != null) {
             writer.writeLine("/// <summary>");
-            this.summary.split("\n").forEach((line) => {
+            this.summary?.split("\n").forEach((line) => {
```

Comment:
Can we drop the `?.` guard if we already have the `this.summary != null` guard above?